site/form.js

```diff
@@ -0,0 +1,15 @@
+var Product = document.getElementById('product');
+var ProductMessage = document.getElementById('productMessage');
+
+function productMessage() {
+  var product = this.options[this.selectedIndex].value;
+  if (product !== '') {
```

Comment:
it technically works, but it might not be a great idea to have two names that are so similar (the `ProductMessage` and the function `productMessage`). Could be easy to confuse.  Also, it's a convention in JavaScript that variable that starts with a capital letter is a constructor function. Not required, obviously, but makes it easier for others to read through and understand your code.